### PR TITLE
Move the Performance tab in settings page to user preferences page

### LINKED ISSUE #3891: [UI] Move the Performance tab in settings page to user preferences page
#### Current Behavior
In settings page, we have a tab called _Performance_ which is supposed to set the default configs for load test for that particular user. It makes more sense for it to be in  user preferences page since the load test configs are specific to a user and not to a system.

Screenshot 2021-08-15 at 7 31 17 PM

#### Desired Behavior
Performance tab in settings page should be completed moved to the user preferences page

Screenshot 2021-08-15 at 7 28 57 PM

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by ghost**
Signed-Off-by: Yash Agarwal <yagarwal1307@gmail.com>

This PR fixes #3891

### CODE REVIEW

**Review by sudo-NithishKarthik — CHANGES_REQUESTED** [review]
@yagarwal1307 , a small change, can you revert the changes you made in `MesherySettings.js` file and just comment out the code that display's the Performance Tab ?

**Comment by yagarwal1307** [comment]
@sudo-NithishKarthik cool I will do that!

File: ui/components/MesherySettings.js [hunk 1]
```diff
@@ -326,7 +325,7 @@ class MesherySettings extends React.Component {
                 tab="tabMetrics"
               />
             </Tooltip>
-            <Tooltip title="Choose Performance Test Defaults" placement="top">
+            {/* <Tooltip title="Choose Performance Test Defaults" placement="top">
```
Comment (sudo-NithishKarthik) [inline ui/components/MesherySettings.js:328]: Also, add a comment here that says why you have commented this , that is, _functionality of this tab will take on a different shape, until that, keeping it commented_

**Review by sudo-NithishKarthik — APPROVED** [review]
LGTM
Thank you for your contributions @yagarwal1307 🎉

**Comment by leecalcote** [comment]
We have a bug introduced here. The Remote Provider tab should only be shown when the user is connected to a Remote Provider.

This screenshot is of a user using the Local Provider (None). They should not see the Remote Provider tab.
Screen Shot 2021-08-18 at 7 30 22 PM

**Pull request merged**